pandas/core/series.py

```diff
@@ -861,6 +861,8 @@ def __getitem__(self, key):
 
                 return result
             except InvalidIndexError:
+                if not isinstance(self.index, MultiIndex):
+                    raise
```

Comment:
Is this related to the rest of the PR? Is there a test that otherwise fails?

Reply:
woops, not related, thought i had reverted this